src/pages/products/Product.tsx

```diff
@@ -72,14 +79,34 @@ export default function Product() {
 
     if (!isFormBusy) {
       setErrors(undefined);
-
       setIsFormBusy(true);
 
-      request('PUT', endpoint('/api/v1/products/:id', { id }), productValue)
-        .then(() => {
+      toast.processing();
+
+      axios
+        .all([
+          request(
+            'PUT',
+            endpoint('/api/v1/products/:id', { id }),
+            productValue
+          ),
+          request(
+            'PUT',
+            endpoint('/api/v1/companies/:id', { id: companyChanges?.id }),
+            companyChanges
+          ),
+        ])
+        .then((response) => {
```

Comment:
This is a big no.
Please update the company if needed. Check the example on the invoices edit page.